lib/tasks/demo.rake

```diff
@@ -246,6 +246,8 @@ desc 'Backup the database'
     printf("Generating monitorees...")
     patients = []
     histories = []
+    available_lang_codes = Languages.all_languages.keys.to_a.map(&:to_s)
```

Comment:
actually, can we move this to lines 165 and 195 that way it's outside of the outer loop as well?

Reply:
Sure! Will do.